README.md

```diff
@@ -161,70 +191,109 @@ You must download `envsubst` for your platform from [https://github.com/a8m/envs
     
-    For example for domain `sockshop.mycompany.com` and `memory` backend, use the following 
+    For example for domain `sockshop.mycompany.com` and `core` backend, use the following 
     
     ```bash
     $ export SOCKSHOP_DOMAIN=sockshop.mycompany.com
-    $ export SOCKSHOP_BACKEND=memory                            
+    $ export SOCKSHOP_BACKEND=core                            
     
-    $ envsubst -i k8s/optional/ingress.yaml| kubectl apply -f -
+    $ envsubst -i k8s/optional/ingress.yaml | kubectl apply -n sockshop-${SOCKSHOP_BACKEND} -f -
     
-    $ kubectl get ingress  
+    $ kubectl get ingress -n sockshop-${SOCKSHOP_BACKEND}  
     
-    NAME               HOSTS                                                                                         ADDRESS           PORTS   AGE
-    mp-ingress         mp.memory.mycompany.com                                                                       XXX.XXX.XXX.XXX   80      12d
-    sockshop-ingress   memory.sockshop.mycompany.com,jaeger.memory.sockshop.mycompany.com,api.memory.mycompany.com   XXX.XXX.XXX.XXX   80      12d
+    NAME               HOSTS                                                                                            ADDRESS           PORTS   AGE
+    mp-ingress         mp.core.sockshop.mycompany.com,swagger.core.sockshop.mycompany.com                               XXX.XXX.XXX.XXX   80      12d
+    sockshop-ingress   core.sockshop.mycompany.com,jaeger.core.sockshop.mycompany.com,api.core.sockshop.mycompany.com   XXX.XXX.XXX.XXX   80      12d
     ```         
 
 1. Access the application
 
-    Access the application via the endpoint http://memory.sockshop.mycompany.com/ 
+    Access the application via the endpoint http://core.sockshop.mycompany.com/ 
+
+### Configure Jaeger
+            
+1. Install the Jaeger operator
+ 
+    This is only required once regardless of the number of backends you want to deploy.
+            
+    ```bash
+    $ kubectl create -f k8s/optional/jaeger-operator.yaml 
+    ```
+
+1. Configured Jaeger
+
+    This is required for each back-end namespace.
+    
+    ```bash
+    $ kubectl create -f k8s/optional/jaeger.yaml -n sockshop-${SOCKSHOP_BACKEND}
+    ```                                         
+
+1. Exercise the Application and access Jaeger
+
+    Using the application UI, place an order or two then look at the trace for the `orders` 
+    service using the
+    Jaeger UI at http://jaeger.core.sockshop.mycompany.com/.
+    
+1.  Sample Jaeger Screens
+
+    ![Jaeger Search](./doc/images/jaeger-search.png)   
+
+    ![Jaeger Detail](./doc/images/jaeger-detail.png)     
+
+### Access Swagger
+
+1. Configured Swagger
+
+    This is required for each back-end namespace.
+    
+    ```bash
+    $ kubectl create -f k8s/optional/swagger.yaml -n sockshop-${SOCKSHOP_BACKEND}
+    ```    
+   
+   Access the Swagger UI at http://swagger.core.sockshop.mycompany.com                                    
```

Comment:
Access the Swagger UI at http://mp.core.sockshop.mycompany.com/swagger/.

Enter `/carts/openapi` into the `Explore` field at the top of the screen and click on `Explore` button. You should see the screen similar to the following, showing you all the endpoints for the `Carts` service (and their payloads), and allowing you to make API requests to it directly from your browser.

(take a screenshot and insert image here)